### ISSUE TITLE
Broader sharing

### ISSUE THREAD

**Issue by felix-lessoer** [alexfrancoeur/kibana_canvas_examples#8]
@alexfrancoeur would you mind if I publish your great canvas examples in my [Kibana Canvas examples](https://elastic-content-share.eu/downloads/category/kibana/kibana-canvas-examples/) gallery ?

**Comment by alexfrancoeur** [comment]
Hey @felix-lessoer, so sorry for the delayed response! Yes by all means, please feel free to use these as you'd like. Just note that I have not updated this repo in quite some time, so some examples may be a bit outdated

**Comment by felix-lessoer** [comment]
Great thank you!
btw: Is Canvas still under active development? It feels like there was not much new stuff during the last releases..

**Comment by alexfrancoeur** [comment]
Yes! Canvas is still in active development. I believe this project board can give you some insight into where the team is currently focused: https://github.com/elastic/kibana/projects/38

Feel free to drop into the #kibana-canvas channel on slack as well

**Comment by nb71** [comment]
I think it will be very useful to have an updated sample gallery using beats data

**Comment by felix-lessoer** [comment]
I'm working on it
